src/core/auth.rs

```diff
@@ -155,4 +155,13 @@ impl Auth {
             })
         }
     }
+
+    pub async fn email_exists(mongo_client: &Client, email: &str) -> Result<bool> {
```

Comment:
Why we are building another function? When what it is just doing is calling another function inside? @mukeshkuiry

Reply:
Removed